authenticationFailure="Authentication Failure in connecting to xtremio array using {0}."
resourceNotFound="Resource {0} not found";
internalError="Internal Error : {1}";
internaljsonParserError="Json Parser Error";
discoveryFailed="Discovery failed for xtremio system {0}";
scanFailed="Scan failed for xtremio system {0}";
moreThanOneClusterNotSupported="XtremIO {0} configured with one more than one cluster which is not supported."
operationNotSupportedForVersion="Operation {0} not supported for the xtremio array version"
noConnectionFound = Operation failed due to failure in getting Xtremio client for url {0}. 